app/src/main/res/values/strings.xml

```diff
@@ -46,24 +46,25 @@
     <string name="sendLabel">Send…</string>
     <string name="editCardTitle">Edit Card</string>
     <string name="addCardTitle">Add Card</string>
-    <string name="scanCardBarcode">Scan Card Barcode</string>
+    <string name="scanCardBarcode">Scan Barcode</string>
     <string name="cardShortcut">Card Shortcut</string>
     <string name="noCardsMessage">Add a card first</string>
-    <string name="card_ids_copied">Copied Card ID(s)</string>
-    <string name="barcodeImageDescriptionWithType">Image of card barcode of type <xliff:g>%s</xliff:g></string>
+    <string name="card_ids_copied">Copied ID(s)</string>
+        <!-- %s is a type of barcode here -->
+    <string name="barcodeImageDescriptionWithType">Image <xliff:g>%s</xliff:g> barcode</string>
     <string name="noStoreError">No name entered</string>
-    <string name="noCardIdError">No card ID entered</string>
-    <string name="noCardExistsError">Could not find card</string>
+    <string name="noCardIdError">No ID entered</string>
+    <string name="noCardExistsError">Could not find that card</string>
     <string name="failedParsingImportUriError">Could not parse the import URI</string>
     <string name="importExport">Import/Export</string>
     <string name="exportName">Export</string>
-    <string name="importExportHelp">Backing up your cards allows you to move them to another device.</string>
+    <string name="importExportHelp">Backing up your data allows moving it to another device.</string>
     <string name="importSuccessfulTitle">Imported</string>
     <string name="importFailedTitle">Import failed</string>
-    <string name="importFailed">Could not import cards</string>
+    <string name="importFailed">Could not perform import</string>
     <string name="exportSuccessfulTitle">Exported</string>
     <string name="exportFailedTitle">Export failed</string>
-    <string name="exportFailed">Could not export cards</string>
+    <string name="exportFailed">Could not perform export</string>
     <string name="importing">Importing…</string>
     <string name="exporting">Exporting…</string>
     <string name="noExternalStoragePermissionError">Grant external storage permission to import or export cards first</string>
```

Comment:
Was thinking "data first", but too vague.

Reply:
I mean, why not:

"Grans external storage permission to import or export data"

Because you import and export cards, but also groups and images and all. The whole "first" word can be dropped.

Reply:
Dropped "first" where it didn't do much.